cmd/launchBackup.go

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
```

Comment:
nit: Update copyright dates

Reply:
The work ideally started in last year, but yeah, can update